config.toml

```diff
@@ -1,3 +1,4 @@
+baseURL = "https://ipfs.io/blog"
```

Comment:
@alanshaw what if instead of using `baseURL` you use another key for the RSS template, so that the rest of the pages stay relative to each other?